app/javascript/components/admin/AdminTable.js

```diff
@@ -603,7 +622,7 @@ class AdminTable extends React.Component {
       <div className="mx-2">
         <h1 className="sr-only">Admin Dashboard</h1>
         <Row className="mb-2">
-          <Col className="mb-1">
+          <Col xl={12} className="mb-1">
```

Comment:
I know this wasn't specifically a change you made, but since we're working on the layout of the filters/buttons above the admin table, can we change the `Row` className to `mb-1`, and both of these `Col` classNames to `mb-2`? This will result in the margins/padding looking more uniform for smaller screen sizes and it will still remain the same for the larger size

Before:
<img width="1008" alt="Screen Shot 2021-03-16 at 10 38 40 AM" src="https://user-images.githubusercontent.com/9956561/111327477-ccb0c600-8643-11eb-9080-8fb4516e7973.png">

After:
<img width="1020" alt="Screen Shot 2021-03-16 at 10 35 32 AM" src="https://user-images.githubusercontent.com/9956561/111327389-ba368c80-8643-11eb-899c-1e9ad1104e2b.png">

Reply:
done here: https://github.com/SaraAlert/SaraAlert/pull/756/commits/0d6f70ccdae7e8206715612b46568a233563b59d